### DISCUSSION TITLE
WiFi Password for SmartSpin2k in firmware V3 changed?

### DISCUSSION THREAD

**Discussion by ColoSpin** [doudar/SmartSpin2k#541] [Q&A] [answered]
I updated my firmware and everything went fine.  Except after the update I cant login to the SmartSpin2k network:  It rejects the password "password".  I have tried password. "password", "password, password", etc.   Does anyone have a suggestion?

**Comment by doudar** [comment]
Use the app to update both the device name and ssid to something different (from what they are now) such as SmartSpin2K if they are currently SmartSpin2k . Then reboot, and you can connect to the WiFi AP

**Comment by doudar** [comment, marked as answer]
You can find the Companion App on the [App Store](https://apps.apple.com/us/app/smartspin2k-companion-app/id6477836948)  for Apple devices, otherwise for Android, you'll have to side load it from here: https://github.com/doudar/SS2kConfigApp/releases/tag/1.0.0

**Reply by ColoSpin** [reply]
The app is awesome!!  The SmartSpin2k is worth much more than its price!

**Reply by doudar** [reply]
Awesome! I’m glad you like it! We can always use more social media support - it’s tough competing for bandwidth with big brands. Have a fantastic day!
